pkg/system/phase4_configuring.go

```diff
@@ -426,7 +429,12 @@ func (r *Reconciler) ReconcileDefaultBackingStore() error {
 		}
 	} else {
 
-		return nil
+		if time.Since(r.NooBaa.CreationTimestamp.Time).Minutes() < 2 {
```

Comment:
would take it out to a constant, wait_time_for_backingstore (or something a bit better :) )